ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml

```diff
@@ -18,7 +18,7 @@ resources:
       memory: 250Mi
 tests:
 - as: e2e
-  skip_if_only_changed: ^docs/|^\.github/|^\.tekton/|\.md$|\.adoc$|\.properties$|^(?:.*/)?(?:\.gitignore|.codecov.yml|.yamllint|OWNERS|PROJECT|LICENSE)$
+  skip_if_only_changed: ^doc/|^\.github/|\\.(md|adoc)$|^(?:.*/)?(?:\.gitignore|.codecov.yml|.yamllint|sonar-project.properties|OWNERS|PROJECT|LICENSE)$
```

Comment:
I just realized, could you please also remove .github? 
the directory contains the part that builds the operators and pushes them into quay repo

Reply:
if somebody modifies the github action workflows , it will just be a workflow change why do you want to run e2e on it